### PR TITLE
Moves string overlays to use images.

### PR DESCRIPTION
**Pull request by ComicIronic**
After the bugfix of #10655, I've decided to move as many of our current string overlays as possible over to the image system. This is tested for addition and removal.

### CODE REVIEW

**Comment by PsiOmegaDelta** [comment]
I'm unsure if some parts of this code will actually work, depending on how BYOND does image comparison, and as such I wish to double-check.

For example this section:

```
    if(src.stat == 0)
        src.overlays += image(icon = icon, icon_state = "eyes")
    else
        src.overlays -= image(icon = icon, icon_state = "eyes")
```

Note how when one add/remove overlays a new image instance is created each time.

Unless image creation is a special case or images are considered equal when icon/icon states are, this means that the -= operation won't have an effect because the overlay list won't contain the newly created instance (as it is different from the instance that was previously added).

The reason there are no issue with this in the pAI code is because the entire overlay list is first cut (or set to 0 length at least) before the overlay images are changed.

**Comment by PJB3005** [comment]
@PsiOmegaDelta I believe BYOND uses appearances internally for the overlays list, so assuming the appearance would be the same (because they get stored in a way similar to the string tree, internally), that should still work.

**Comment by GinjaNinja32** [comment]
I'm fairly sure overlays/underlays are lists of appearances, and an image with the same icon and icon_state etc has the same appearance value, ie that code _should_ work fine.

**Comment by ComicIronic** [comment]
It does work, I tested it for stuff like the spell HUD.

File: code/ATMOSPHERICS/components/binary_devices/MSGS.dm [hunk 1]
```diff
@@ -224,19 +224,19 @@
...
-		overlays += "node-1"
+		overlays += image(icon = icon, icon_state = "node-1")
 
 	if(node2)
-		overlays += "node-2"
+		overlays += image(icon = icon, icon_state = "node-2")
 
 	if(!(stat & (NOPOWER | BROKEN)))
 
 		overlays += "o-[i]"
```
Comment (clusterfack) [inline code/ATMOSPHERICS/components/binary_devices/MSGS.dm]: damn regex missed these

**Comment by ComicIronic** [comment]
Alright, I updated it with replacements for the embedded strings as well.

**Pull request merged**